rectools/models/implicit_als.py

```diff
@@ -150,7 +143,13 @@ def __init__(self, model: AnyAlternatingLeastSquares, verbose: int = 0, fit_feat
     def _make_config(
         cls, model: AnyAlternatingLeastSquares, verbose: int, fit_features_together: bool
     ) -> ImplicitALSWrapperModelConfig:
-        params = {
+        model_cls = (
+            model.__class__
+            if model.__class__ not in (CPUAlternatingLeastSquares, GPUAlternatingLeastSquares)
+            else "AlternatingLeastSquares"
```

Comment:
Unfortunately, we can't  
Since I specified a `Literal` in the type of model class, it should be this literal explicitly, it couldn't be a variable (I mean, it could, for the code it's fine, but not for the mypy)